docs/index.html

```diff
@@ -203,10 +203,17 @@ <h5>If you have any legal concerns regarding copyrights or want to report an abu
 
       <div class="search">
         <input type="text" placeholder="Search for icons" ng-model="search">
+        <input type="text" class="result" value="{{ filtered.length }}" title="Number of icons in the current search" disabled>
```

Comment:
Since the total of icons is written bellow, I think it's possible to delete this text input.
```suggestion
```

Reply:
I disagree. 
The numbers are for different things, and it's not obvious without either hovering over the number, or searching

Reply:
I suggest adding this though :)
```suggestion
        <input type="text" class="result" value="{{ filtered.length }}" title="Number of icons in the current search result" disabled>
```

Reply:
Hmm, my bad! 😅
I've got it now!
I'm ok with the previous `title`, I think we can just discard my suggestion here. 🙂